### ISSUE TITLE
Error on log entries without content type

### ISSUE THREAD

**Issue by eviltnan** [barseghyanartur/django-admin-timeline#6]
AttributeError at /timeline/
'NoneType' object has no attribute 'name'

```
            ]
        )
    )
def get_content_types(data):
    """Get content type choices."""
    return list(set([(d.content_type_id, d.content_type.name) for d in data])) ...
class FilterForm(forms.Form):
    """Filter form to be used in the timeline.
    ``users``: Users list to be filtered on.
```

**Comment by barseghyanartur** [comment]
@eviltnan:

Thanks for bringing this up. I'll fix it soon.

**Comment by barseghyanartur** [comment]
@eviltnan:

Fixed in version 1.6.1.
